requirements/constraints.txt

```diff
@@ -0,0 +1,28 @@
+# Version constraints for pip installation.
+#
+# This file doesn't install any packages. It specifies version constraints
+# that will be applied if a package is needed.
+#
+# When pinning something here, please provide an explanation of why.  Ideally,
+# link to other information that will help people in the future to remove the
+# pin when possible.  Writing an issue against the offending project and
+# linking to it here is good.
+
+
+# Test case helper compatible with open-release/ironwood.master
+ddt==1.2.0
```

Comment:
Is this constraint needed? I think that it is only used in the tests we could try to use a newer version

Reply:
Agreed. Constraint removed.